installer/mac/dropDmgConfig/layouts/bracketsLayout/Info.plist

```diff
@@ -46,6 +46,16 @@
 					<key>type</key>
 					<string>file</string>
 				</dict>
+				<dict>
```

Comment:
@gavankar can you verify this layouting change and see if it's correct. The DMG GUI layout is getting messed up on my machine when I simply mount it. The container bounds seem to be off.

![image](https://user-images.githubusercontent.com/34858846/38320646-c89dbbc8-3852-11e8-8de0-b275f90581af.png)

@ayushrathi15 can you confirm this issue?

Reply:
This layout change was added after this bug was found, to fix this bug